pkg/machine/qemu.go

```diff
@@ -156,10 +156,6 @@ func (q *QEMU) Stop() error {
 
 func (q *QEMU) Clean() error {
 	if q.machineConfig.StateDir != "" {
-		if err := q.Stop(); err != nil {
```

Comment:
`Stop()` is already called when we call `vm.Destroy(nil)`, no need to call it here too